app/assets/javascripts/legacy/views/report/edit_view.js

```diff
@@ -156,43 +160,59 @@
 
   // applies a given function to all panes
   klass.prototype.pane_do = function (func_name) {
-    for (let i = 0; i < this.panes.length; i++) if (this.panes[i][func_name]) this.panes[i][func_name](Array.prototype.slice.call(arguments, 1));
```

Comment:
You didn't like my 146 char line that I wrote in 2012? :P

Reply:
Fabulously dense 👏